core/bessctl.cc

```diff
@@ -534,6 +534,9 @@ class BESSControlImpl final : public BESSControl::Service {
     }
 
     for (; i <= wid_filter; i++) {
+      if (workers[i] == nullptr) {
```

Comment:
I am not sure if I understood the code correctly, but it seems like if:
* `wid_fillter == -1` (all workers) specified by the client
* `num_workers == 2` and and there are workers with `wid == 1` and `wid ==2` (i.e., no worker with `wid == 0`)
then the `for` loop will iterate only over `i == 0` and `i == 1`, missing the worker with `wid == 2`..?